WordPress/Classes/Models/Theme.m

```diff
@@ -57,6 +57,15 @@ + (Theme *)createOrUpdateThemeFromDictionary:(NSDictionary *)themeInfo
     return theme;
 }
 
+#pragma mark - CoreData helpers
+
++ (NSString *)entityName
+{
+    return NSStringFromClass([self class]);
```

Comment:
Unfortunately some of our classes don't have the same entity name and class name (such as in `WPAccount` vs `Account`).  For the scope of this particular PR I'd avoid going into that analysis - if the time comes when we decide to unify this, we can also remove this method.